queries/hack/highlights.scm

```diff
@@ -0,0 +1,295 @@
+(variable) @variable
+(identifier) @variable
+((variable) @variable.builtin
+ (#eq? @variable.builtin "$this"))
+
+(braced_expression) @none
+
+(scoped_identifier
+ (qualified_identifier 
+   (identifier) @type))
+
+(comment) @comment
+(heredoc) @comment
+
+[
+ "function"
+] @keyword.function
+
+[
+ "async"
+ "await"
+ "type"
+ "interface"
+ "implements"
+ "class"
+ "protected"
+ "private"
+ "public"
+ "using"
+ "namespace"
+ "attribute"
+ "const"
+ (xhp_modifier)
+ (final_modifier)
+ "extends"
+  "insteadof"
+] @keyword
+
+"use" @include
+
+[
+  "new"
+  "print"
+  "echo"
+  "newtype"
+  "clone"
+  "as"
+] @keyword.operator
+
+[
+ "return"
+] @keyword.return
+
+[
+  "shape"
+  "tupe"
+  (array_type)
+  "bool"
+  "float"
+  "int"
+  "string"
+  "arraykey"
+  "void"
+  "nonnull"
+  (null)
```

Comment:
I don't know hack, but is this a null type, or the constant `null` ?

Reply:
Fixed it to constant.builtin.